### PR TITLE
feat add AirPlayIconButton widget, it use a default IconTheme

### PR DESCRIPTION
**Pull request by SimoneBressan**
- feat add AirPlayIconButton widget, it use a default IconTheme
- feat add onShowPickerView and onClosePickerView to AirPlayRoutePickerView
- Arranged the files inside the library and exposed a name for the library

### CODE REVIEW

File: ios/Classes/FlutterRoutePickerView.swift [hunk 1]
```diff
@@ -12,45 +12,67 @@ import Flutter
...
-                tempView.backgroundColor = UIColor.init(red: color["red"] as! CGFloat,
-                                                  green: color["green"] as! CGFloat,
-                                                  blue: color["blue"] as! CGFloat,
-                                                  alpha: color["alpha"] as! CGFloat)
+                tempView.backgroundColor = FlutterRoutePickerView.mapToColor(color)
             }
             
+            if #available(iOS 13.0, *) {
+                tempView.prioritizesVideoDevices = arguments["prioritizesVideoDevices"] as! Bool
+            }
```
Comment (GabM3) [inline ios/Classes/FlutterRoutePickerView.swift:39]: else?? nothing?
```suggestion
            if #available(iOS 13.0, *) {
                tempView.prioritizesVideoDevices = arguments["prioritizesVideoDevices"] as? Bool ?? false
            }
```

**Review by GabM3 — CHANGES_REQUESTED** [review]

**Comment by MrJai** [comment]
That's good work @SimoneBressan. Thanks for your contribution.
I will merge and upload a new version in a day.

**Comment by vanlooverenkoen** [comment]
@SimoneBressan any idea when you could pick up this pull request?

**Comment by roman800** [comment]
Is it only the changelog.md that is missing for this pr to be complete ? 
Could really use the onShowPickerView and onClosePickerView for the AirPlayRoutePickerView for my project 🙏

**Comment by SimoneBressan** [comment]
@MrJai Updated Changelog and fixed color management

**Comment by MrJai** [comment]
I will review and release a new version today.

**Comment by MrJai** [comment]
The new version is published. Thank you @SimoneBressan for your contribution!

**Pull request merged**